webapp/javascript/components/Settings/Apps/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from '@webapp/redux/hooks';
+import {
+  selectApps,
+  reloadApps,
+  deleteApp,
+} from '@webapp/redux/reducers/settings';
+import { addNotification } from '@webapp/redux/reducers/notifications';
+import { type App } from '@webapp/models/app';
+import Input from '@webapp/ui/Input';
+import TableUI from '@webapp/ui/Table';
+import cl from 'classnames';
+
+import appsStyles from './Apps.module.css';
+import tableStyles from '../SettingsTable.module.scss';
+import { getAppTableRows } from './getAppTableRows';
+
+const headRow = [
+  { name: '', label: '', sortable: 0 },
+  { name: '', label: 'Name', sortable: 0 },
+  { name: '', label: '', sortable: 0 },
+];
+
+function Apps() {
+  const dispatch = useAppDispatch();
+  const apps = useAppSelector(selectApps);
+  const [search, setSearchField] = useState('');
+
+  useEffect(() => {
+    dispatch(reloadApps());
+  });
```

Comment:
I tried running this and currently it gets into an infinite loop. I think maybe there needs to be an empty array here as the second argument or something like that (not a big expert in react so I'm not sure)

![Screen Shot 2022-08-19 at 2 58 19 AM](https://user-images.githubusercontent.com/662636/185594955-acfe7659-a05b-4702-9030-80c385ff1ef2.png)

Reply:
Addressed in https://github.com/pyroscope-io/pyroscope/pull/1424/commits/66cc28df6dc973c3295e501e65807ebd888b5195